claims-provider-archetype/src/main/resources/archetype-resources/src/main/java/__pluginName__/__pluginName__ClaimsProvider.java

```diff
@@ -0,0 +1,31 @@
+package ${package}.${pluginName};
+
+import se.curity.identityserver.sdk.attribute.Attributes;
+import se.curity.identityserver.sdk.attribute.AuthenticationAttributes;
+import se.curity.identityserver.sdk.claims.ClaimsProvider;
+import se.curity.identityserver.sdk.claims.RequestedClaimAttribute;
+import se.curity.identityserver.sdk.data.authorization.ScopeValue;
+import se.curity.identityserver.sdk.oauth.OAuthClient;
+
+import java.util.Set;
+import sun.reflect.generics.reflectiveObjects.NotImplementedException;
+
+public final class ${pluginName}ClaimsProvider implements ClaimsProvider
+{
+    private final ${pluginName}ClaimsProviderConfig _configuration;
+
+    public ${pluginName}ClaimsProvider(${pluginName}ClaimsProviderConfig configuration) {
+        _configuration = configuration;
+
+        // See examples of Claims Providers on GitHub: https://github.com/search?q=topic%3Aclaims-provider+org%3Acurityio
+    }
+
+    @Override
+    public Attributes getClaimValues(Set<RequestedClaimAttribute> requestedClaimAttributes,
+        Set<ScopeValue> scopeValues,
+        AuthenticationAttributes userAuthenticationAttributes,
+        OAuthClient client)
+    {
```

Comment:
we should be consistent in formatting; we don't have to use our style necessarily, but we shouldn't mix styles within the same file

Reply:
It's hard to style automatically those files as IntelliJ doesn't recognise properly the template variables used in method or class names. I'll try to go once again through the files and change formatting.

Reply:
The formatting should now be consistent.

Reply:
Understandable. It'll be easy for the author to reformat as well, so let's not loose sleep over this.